server/test/server.spec.js

```diff
@@ -3,24 +3,138 @@ const request = require('supertest');
 const express = require('express');
 const expect = require('chai').expect;
 const app = require('../app.js');
+const dbUtils = require('../../db/lib/utils.js');
 
-describe('basic server', function() {
-  it('sends back hello world', function(done) {
+describe('Unauthorized Routes', function() {
+  it('redirects to login if not authed', function(done) {
     request(app)
-      .get('/api')
+      .get('/dashboard')
+      .expect(302)
+      .expect(function(res) {
+        expect(res.header.location).to.equal('/login');
+      })
+      .end(done); 
+  });
+
+  it('serves / without auth', function(done) {
+    request(app)
+      .get('/')
       .expect(200)
       .expect(function(res) {
-        expect(res.text).to.equal('Hello World!');
+        expect(res.text).to.include('<script src="/dist/bundle.js"></script>');        
```

Comment:
you'll probably want to use this setting set to true in vscode so we dont get trailing spaces as a diff item in the future. although in this PR it's not creating this problem.

`"files.trimTrailingWhitespace": true`